pkg/3scale/amp/component/system.go

```diff
@@ -243,7 +243,6 @@ func (system *System) buildSystemBaseEnv() []v1.EnvVar {
 
 	if system.Options.S3FileStorageOptions != nil {
 		result = append(result,
-			helper.EnvVarFromConfigMap("FILE_UPLOAD_STORAGE", "system-environment", "FILE_UPLOAD_STORAGE"),
```

Comment:
Thanks for spotting this issue. I have checked history and it has been there for quite a long time. 

We should not add this change in this PR. It deserves a its own PR.

Reply:
Opened a new PR for this change (#811) and removed the commit.